### DISCUSSION TITLE
Find bug and got banned :)

### DISCUSSION THREAD

**Discussion by Ntrglhf** [chromiecraft/chromiecraft#2280] [General]
I came across a bug in Silithyst. On the side of the alliance, when trying to remove sand from itself, it is not removed, but honor and reputation are awarded. However, when trying to record a bug, I got a possible ban for further placement, by repeated repetition of the error for clarity. The playable character who was exposed to the bug was Onlytoday. 

p.s. After checking the bug tracker, I saw that a similar problem had already been entered, but from the side of the horde.

**Comment by 55Honey** [comment]
Hi Ntrglhf,
You never ever want to intentionally use a bug on ChromieCraft. Simple reason: GMs can not tell the intentions of the one doing it. 

There is no choice but to leave your 7day ban in place.

**Comment by Amrx2** [comment]
Hi Ntrglhf, whilst I can understand you wanted to check this and record, I watched you for a good 30 seconds going in and out farming honor and rep. Popping in 2-3 times but have been sufficient to confirm it was bugged, this wasn't what you were doing. 

Do you have the recording to validate your claims? Thanks, Construct.
